adjusts joke_accent

kyoniichan opened the pull request:
setup help correctly
made it so that bot message is in the same format as the random repeating function
bot also deletes the message that gives the command

kyoniichan committed:
adjusts joke_accent

setup help correctly
made it so that bot message is in the same format as the random repeating function
bot also deletes the message that gives the command

CaptSeagull reviewed (changes requested):
line 336 - converted isn't used anymore. You can remove if needed.
line 337 - errors out on phrase as there's no variable for it. I recommend using string from line 335
line 344 - don't forget to replace "Hehe..." with title variable
line 345 - it errors on ctx.message.author.user.avatar_url. Double checked, and it should be just ctx.message.author.avatar_url
line 347 - this line won't be reached. I recommend to put this before line 346. Also remove the return so should only be await self.bot.delete_message(ctx.message)

kyoniichan committed:
resolves issues

this should fix the requested changes
"line 336 - converted isn't used anymore. You can remove if needed.
line 337 - errors out on phrase as there's no variable for it. I recommend using string from line 335
line 344 - don't forget to replace "Hehe..." with title variable
line 345 - it errors on ctx.message.author.user.avatar_url. Double checked, and it should be just ctx.message.author.avatar_url
line 347 - this line won't be reached. I recommend to put this before line 346. Also remove the return so should only be await self.bot.delete_message(ctx.message)"

kyoniichan reviewed (commented):
fixed

CaptSeagull reviewed (approved):
Works now, thanks! Very good.
